include/contactPageSQLinsert.php

```diff
@@ -1,5 +1,6 @@
 <?php
-include('config/init.php');
+function contactQuery($name, $email, $comment){
     dbQuery("INSERT INTO contactPage (name, email, comment)
-        VALUES (:name, :email, :comment)", array('name'=>$_REQUEST['name'], 'email'=>$_REQUEST['email'], 'comment'=>$_REQUEST['comment']));
+        VALUES (:name, :email, :comment)", array('name'=>$name, 'email'=>$email, 'comment'=>$comment));
                 echo echoHeaderHtml("Contact", "More Information about Me")." <br/></br><h4 class='contact'> Message successfully recorded! </h4><br/></br>". echoFooterHtml();
```

Comment:
I sort of mentioned this above, but in functions like these, I'd pull the html out and only do the query. In the case of insert queries, it's nice to return the Id of the record you just inserted in case the code calling this function wants to know it.

But print the html somewhere else. In this case you want to redirect to another page and then print the confirmation message there.